src/common/ceph_context.cc

```diff
@@ -391,6 +391,12 @@ void CephContext::do_command(std::string_view command, const cmdmap_t& cmdmap,
   }
   lgeneric_dout(this, 1) << "do_command '" << command << "' '"
 			 << ss.str() << dendl;
+  if (command == "assert") {
```

Comment:
I worry about how dangerous these are with no confirmation.  Also, why don't any of these generic commands show up in "ceph daemon <> help"?

Reply:
Yeah, I waffled on whether to leave this patch DNM.  Maybe if we guard it by a config option debug_allow_asok_abort_and_assert?